index.html

```diff
@@ -0,0 +1,348 @@
+          been the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-skil button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+      </div>
+
+      <div class="desk-card-d2">
+        <h2 class="desk-prof-art-dash">Data Dashboard Healthcare</h2>
+        <p class="desk-p22-d">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+          been the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-skil button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+
+      </div>
+
+      <div class="desk-card-w1">
+        <h2 class="desk-prof-art-w">Website Protfolio </h2>
+        <p class="desk-w">A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+          the industry's standard</p>
+        <ul class="desk-skils-d">
+          <li class="desk-button-d button-d">html</li>
+          <li class="desk-skil button-d">bootstrap</li>
+          <li class="desk-skil button-d">Ruby</li>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <section class="section-3">
+    <div class="about-me">
+      <h1 class="h31">About me</h1>
+      <p class="p31-desk">Hello I’m a software developer! I can help you build a product ,feature or
+        website Look through some of my work and experience! If you like what you see and have a project you need coded,
+        don’t hestiate to contact me.</p>
+      <a class="get-myresume button" href="">Get My Resume</a>
+    </div>
+    <hr class="line-3">
+    <div class="about-me-desk">
+      <div class="story-abt-me">
+        <h1 class="h31-desk">About me</h1>
+        <p class="p31-desk">Hello I’m a software developer! I can help you build a product ,feature or
+          website Look through some of my work and experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.</p>
+        <a class="get-myresume button" href="">Get My Resume</a>
+      </div>
+      <img class="abt-img-desk" src="./assets/deskimage/Groupme.png" alt="">
+    </div>
+
+    <ul class="lists">
+      <li class="Lan">Language
+        <ul class="lan-list">
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+      </li>
+
+      <li class="Frame"> Framework
+        <ul class="Frame-list">
+          <li>React.js</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+        </ul>
+      </li>
+      <li class="Sk">Skills
+        <ul class="Sk-list">
+          <li>Database management</li>
+          <li>Version control</li>
+          <li>CLI</li>
+          <li>API design</li>
+          <li>Web development</li>
+        </ul>
```

Comment:
- [ ] As requested by the previous reviewer you should stick to the design as much as possible. Skills should be centered, please implement it.
### Your design
![hissssss](https://user-images.githubusercontent.com/81727988/235227806-3269ef41-88dc-42c1-86f5-47daa3816d1c.png)


### Figma design
![his](https://user-images.githubusercontent.com/81727988/235227731-2059dffc-45c9-45c4-89ea-52959d9b6e0f.png)